libpod/runtime.go

```diff
@@ -54,8 +56,11 @@ var (
 			"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
 		},
 		CgroupManager:  "cgroupfs",
+		ExitsDir:       "/var/run/libpod/exits",
```

Comment:
This should be a constant and allowed to be overridden

Reply:
Changed in a later commit